Sources/BraintreeThreeDSecure/BTProxyCardinal.h

```diff
@@ -25,6 +31,7 @@ typedef NS_ENUM(NSUInteger, BTProxyCardinalResponseActionCode) {
 
 @protocol BTProxyCardinalSessionConfiguration <NSObject>
 @property (nonatomic, assign) BTProxyCardinalSessionEnvironment deploymentEnvironment;
+@property (nonatomic, strong) id uiCustomization;
```

Comment:
random question: would it make sense for `uiCustomization` be a `BTThreeDSecureV2UICustomization` instead of `id` here?

Reply:
No, it needs to be `id`, unfortunately. At runtime it will be a Cardinal class, but to work around not having an xcframework for Cardinal, we can't use Cardinal classes at compile time, which is why we're using `id` here.